scalableminds/webknossos
# Cleanup zarr directory representation

This PR
* simplifies the zarr directory html pages
* makes them compatible with python fsspec
* and ensures that the pages are used in the browser always with a trailing slash 

### URL of deployed dev instance (used for testing):
- https://htmldirectoryfsspeccompat.webknossos.xyz

### Steps to test:
I tested those myself, review should be enough:
- Go to https://htmldirectoryfsspeccompat.webknossos.xyz/data/zarr/sample_organization/ROI2017_wkw?token=…,
  and explore some folders (token needs to be re-added manually)
- For the python client:
```python
from upath import UPath

p = UPath(
    "https://htmldirectoryfsspeccompat.webknossos.xyz/data/zarr/sample_organization/ROI2017_wkw",
    client_kwargs={"headers": {"X-Auth-Token": "…"}}
)

p.is_dir()
# True

list(p.iterdir())
# [
#   HTTPPath('https://htmldirectoryfsspeccompat.webknossos.xyz/data/zarr/sample_organization/ROI2017_wkw/color'),
#   HTTPPath('https://htmldirectoryfsspeccompat.webknossos.xyz/data/zarr/sample_organization/ROI2017_wkw/segmentation')
# ]

list(p.iterdir())[0].is_dir()
# True
```

### Issues:
- fixes #

------

## TODO
